canonical-web-and-design/jaas-dashboard
# Add stylelint ✨

## Done

Add stylelint ✨

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Ensure CI tests pass
